src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const clickEvent = (newTab) => {
```

Comment:
```suggestion
  const handleClick = (newTab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364